### ISSUE TITLE
Alert sheets should truncate overly long error messages

### ISSUE THREAD

**Issue by GoogleCodeExporter** [salvatorelai/svnx#145]
```
My company uses the check-style subversion hook.  If a commit results in lots 
of errors the modal window that appears extends beyond the bottom of my 
desktop.  You can't scroll this window, or close it.  The only thing you can do 
is force svnx to quit.  A scroll bar would be nice.
```

Original issue reported on code.google.com by `gmparker...@gmail.com` on 26 Aug 2011 at 2:53

**Comment by GoogleCodeExporter** [comment]
```
You should report that to Apple as that’s really a Cocoa bug.
[Cocoa determines the size (height) of the alert.  It should truncate overly 
long text (or make it scrollable).]

However, you can still close the alert sheet.
Typing <return> or <enter> maps to the OK button & <escape> or ⌘-. maps to 
Cancel (if there is one).

I will, however, also look at making svnX truncate overly long error/alert 
messages,
especially as if you use the Review & Commit window then the commit error is 
also displayed in (the diff pane of) that window.
```

Original comment by `chris...@gmail.com` on 26 Aug 2011 at 4:47
- Changed title: **Alert sheets should truncate overly long error messages**
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Cornerstone does a nice job handling it.  They show the error very similarly to 
SVNX but the popup has a fixed size.  The error is shown in a scrollable text 
field (see attached image)
```

Original comment by `gmparker...@gmail.com` on 29 Aug 2011 at 1:38

Attachments:
- [Screen Shot 2011-08-29 at 10.28.12 AM.png](https://storage.googleapis.com/google-code-attachments/svnx/issue-145/comment-2/Screen Shot 2011-08-29 at 10.28.12 AM.png)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in svnX 1.3.2.
```

Original comment by `chris...@gmail.com` on 1 Jan 2012 at 7:07
- Changed state: **Done**
